### ISSUE TITLE
Valo theme Grid styles in FW8

### ISSUE THREAD

**Issue by heruan** [vaadin/framework#8514]
`Table` was moved to compatibility package in FW8, in favor of `Grid`. Valo, which is still the main theme, has many useful styles for `Table`, such as:

- `ValoTheme.TABLE_NO_STRIPES`
- `ValoTheme.TABLE_NO_VERTICAL_LINES`
- `ValoTheme.TABLE_NO_HORIZONTAL_LINES`
- `ValoTheme.TABLE_BORDERLESS`
- `ValoTheme.TABLE_COMPACT`
- `ValoTheme.TABLE_SMALL`

These style would be really useful also for `Grid`.

**Comment by pleku** [comment]
Can be introduced even in a maintenance version, but we are currently targeting 8.1 to have Grid replace Table completely, with all missing features from Table implemented.

 ping @jouni

**Comment by jouni** [comment]
Yeah, definitely worth adding. I can provide guidance/mentoring for the work, once the dev team is ready to start working on this.

**Comment by heruan** [comment]
Any update on this? I tried a naïve approach by renaming the styles from `table` to `grid`, but that didn't work.

**Comment by berndhopp** [comment]
worth adding, I agree. Grid is not on feature-parity with table unless we have the same built-in styling capabilities.

**Comment by Mi-tweber** [comment]
Also waiting for this, would be helpful

**Comment by djerman** [comment]
Any news on this?

**Comment by ShawnRG** [comment]
Update?
